app-admin/passook/passook-20121001.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit prefix
+
+DESCRIPTION="Password generator capable of generating pronounceable and/or secure passwords"
+HOMEPAGE="https://github.com/mackers/passook"
+# snapshot of https://github.com/mackers/passook.git
+SRC_URI="mirror://gentoo/${P}.tar.gz"
```

Comment:
This is surely a dead `SRC_URI` by now as it will have been purged from our mirrors long ago?

Reply:
I would fetch a commit-based snapshot from github.